File: runtime/src/main/codestarts/quarkus/quarkiverse-github-action-codestart/java/src/main/java/org/acme/MyAction.java
package org.acme;

import io.quarkiverse.githubaction.Action;
import io.quarkiverse.githubaction.Commands;

public class MyAction {

    @Action
    void action(Commands commands) {
        commands.notice("Hello from Quarkus GitHub Action");

        commands.appendJobSummary(":wave: Hello from Quarkus GitHub Action");
    }
}

File: runtime/src/main/codestarts/quarkus/quarkiverse-github-action-codestart/kotlin/src/main/kotlin/org/acme/MyAction.kt
package org.acme;

import io.quarkiverse.githubaction.Action;
import io.quarkiverse.githubaction.Commands;

class MyAction {

    @Action
    fun action(commands: Commands) {
        commands.notice("Hello from Quarkus GitHub Action");

        commands.appendJobSummary(":wave: Hello from Quarkus GitHub Action");
    }
}
